README.md

```diff
@@ -1 +1,191 @@
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Use of linters]**
+- **[Github & Git]**
+- **[README.md file]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [No Live Demo Required](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Node installed in computer
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd desired-folder
+  git https://github.com/Roathy/Hello---Microverse.git
+```
+
+### Install
+
+Install this project with:
+
+- No installation required
+
+### Usage
+
+To run the project, execute the following command:
+
+Open index.html in browser
+
+### Run tests
+
+To run tests, run the following command:
+
+- No testing implement yet
+
+### Deployment
+
+You can deploy this project using:
+
+GitHub can be used to deploy website. For more information about publishing sources, see "About GitHub pages".
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@Roathy](https://github.com/Roathy)
+- Twitter: n/a
+- LinkedIn: [LinkedIn](https://linkedin.com/in/julio-vergara-roathy)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[Desktop version]**
+- [ ] **[Details section]**
+- [ ] **[Responsiveness ]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Roathy/Hello---Microverse/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give it a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse team.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Kindly add the correct path of the MIT `LICENSE` in your project, as I can see you have not added the path correctly.  Please look at the below suggestion to solve your problem.

``` suggestion
This project is [MIT](./MIT.md) licensed.
```